programs/server/config.xml

```diff
@@ -1113,16 +1113,82 @@
         events - send data from table system.events
         asynchronous_metrics - send data from table system.asynchronous_metrics
     -->
-    <!--
+   
```

Comment:
Before this PR the whole `<prometheus>` section in `programs/server/config.xml` was commented out and was much smaller. Let's keep it so.

Reply:
Reverted this change.